_posts/2016-07-20-Automation-testing-with-postman.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+authors: [greg_rinaldi]
+title: 'API Testing with Postman and Newman'
+image: /img/postman.png
+tags: [Postman,Integration Testing,Newman,Automation,Testing]
+category: Testing
+comments: true
+---
+### Prerequisites
+For the purpose of this tutorial it is mandatory to have [Postman](https://www.getpostman.com/){:target="_blank"} installed which is available as a Chrome extension, as well as a Mac application. And it is also mandatory to create an account at [Algorithmia](https://www.algorithmia.com/){:target="_blank"}.
```

Comment:
. And it is also mandatory => It is also mandatory (do not start sentence with And)